extensions/extensions-helm/aissemble-configuration-store-chart/values.yaml

```
@@ -11,6 +11,18 @@ aissemble-quarkus-chart:
       tag: ""
     hostname: configuration-store
     restartPolicy: Always
+    supplementalVolumes:
+      - name: baseconfig-volume
```

```suggestion
      - name: base-config-volume
```
To be consistent with the naming elsewhere.